modules.yml

```
@@ -129,6 +133,10 @@ categories:
         slug: 'micronaut-cassandra'
         title: 'Micronaut for Cassandra'
         description: 'Integration between Micronaut and Cassandra'
+      coherence:
+        slug: 'micronaut-coherence'
+        title: 'Micronaut for Coherence'
+        description: 'Integration between Micronaut and Oracle Coherence.'
```

```suggestion
        description: 'Integration with Oracle Coherence.'
```

From the [trademark guidelines](https://micronaut.io/brand-guidelines/): 

> You may not use the word Micronaut as a verb or a noun. It should be used only as an adjective followed by a noun (e.g., “I love using the Micronaut® framework,” not “I love using Micronaut!”).